### PR TITLE
WIP: Adding recipe for HLA-MA

### PR DESCRIPTION
**Pull request by holtgrewe**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core

Hi, this adds a recipe for the HLA-MA tool.

The code builds fine with `conda build` but fails in Docker. Could someone point me in the right direction to resolve this?

### CODE REVIEW

**Comment by blankclemens** [comment]
Activating the console output in the test command shows an error, maybe that can help:

    + hlama --help
    Traceback (most recent call last):
      File "/anaconda/envs/_test/bin/hlama", line 4, in <module>
        import hlama.app
    ImportError: No module named 'hlama'

**Comment by holtgrewe** [comment]
Thanks, how did you activate it?

**Comment by blankclemens** [comment]
i just didn't put the output on `dev/null` ;) :

`hlama --help` instead of `hlama --help &>/dev/null`

But I don't know how to interpret this error - building with conda works fine, I can confirm that.

File: recipes/hlama/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  fn: hlama-develop.tar.gz
+  url: https://github.com/bihealth/hlama/archive/develop.tar.gz
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - snakemake ==3.7.1
+  run:
+    - snakemake ==3.7.1
```
Comment (bgruening) [inline recipes/hlama/meta.yaml]: Runtime dependency on python is missing.

File: recipes/hlama/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package:
+  name: hlama
+  version: "0.1"
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/hlama/meta.yaml]: New packages should start with 0

**Comment by holtgrewe** [comment]
OK, thanks for the help.

I have created a patch for FRED-2/OptiType#36 to make it support Python 3. If this is committed, I'll include an update for the OptiType recipe and then directly depend on this so everything is in Python 3 and dandy. Otherwise, I'll think of something...

**Comment by bgruening** [comment]
@holtgrewe is this ready to go?

**Comment by holtgrewe** [comment]
@bgruening as optitype now supports Python 3, let me simplify the installation a bit first

File: recipes/hlama/meta.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 package:
   name: hlama
-  version: "0.2"
+  version: '0.3'
```
Comment (bgruening) [inline recipes/hlama/meta.yaml]: Can you please use a stable URL like: https://github.com/bihealth/hlama/archive/v0.3.tar.gz

**Comment by holtgrewe** [comment]
Hm, I don't understand the problem here. Something in the dependency chain does not support Python 3.5. I think the problem is coming from installation on Python 3.5... What can I do to fix this problem?

https://travis-ci.org/bioconda/bioconda-recipes/jobs/159298139

**Comment by bgruening** [comment]
@holtgrewe the reason is that pyomo is not available for python 3.5. One way would be to update pyomo and build it for python 3.5 as well. But this involves more work as you see here: https://github.com/bioconda/bioconda-recipes/pull/2312

Feel free to improve this branch.

**Comment by holtgrewe** [comment]
Could I somehow skip the 3.5 build for now so we can merge this now and then upgrade pyomo somehow?

**Comment by bgruening** [comment]
You can add a skip: True #[py35]
On Sep 14, 2016 8:22 AM, Manuel Holtgrewe <notifications@github.com> wrote:Could I somehow skip the 3.5 build for now so we can merge this now and then upgrade pyomo somehow?

File: recipes/hlama/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+package:
+  name: hlama
+  version: '0.3'
+
+build:
+  number: 0
+  skip: True # [py27 or py35 or osx]
```
Comment (holtgrewe) [inline recipes/hlama/meta.yaml]: @bgruening I tried and it did not work. Can you help here? Thanks

Comment (bgruening) [inline recipes/hlama/meta.yaml]: Try to skip py3k. I will look at it in the evening.

Comment (holtgrewe) [inline recipes/hlama/meta.yaml]: Ah, HLA-MA only works for Python 3 because it depends on Snakemake.

No worries, no hurry. Thanks! 🙇

Comment (holtgrewe) [inline recipes/hlama/meta.yaml]: ping, any idea?

**Pull request closed without merging**